ics/src/main/java/com/example/demo/controllers/ImagesController.java

```diff
@@ -78,12 +84,13 @@ public void deleteImage(@PathVariable Long id) {
     }
 
     @RequestMapping(value = {"id"}, method = RequestMethod.PUT)
```

Comment:
I think that this mapping will not work, the correct mapping should be `value = {"/{id}"}`

Reply:
The same applies to `deleteImage` endpoint

Reply:
adjusted

Reply:
adjusted